### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by abiodunraheem**
## Portfolio: details popup window
- I used JavaScript to manipulate DOM elements. javascript events.
- parse a Figma design to create a UI.
- Used objects to store and access the cards data.

### CODE REVIEW

**Review by sja-thedude — COMMENTED** [review]
## Invalid Code Review Request ❌

Hi @abiodunraheem, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have commits from all the members in your pair programming group. Kindly make sure to have commits from all members in your pair programming group before submitting it for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 1]
```diff
@@ -410,6 +216,36 @@ <h2>About me</h2>
         <hr class="mobile-hr">
       </div>
     </div>
+
+    <div class="popup-window" id="pop-window"></div>
```
Comment (Mre55) [inline index.html]: Nice job with the popup window 👍. Now let's try to make it `popup` window centered currently it's off to the left. And also you display the popup window let's try to make the background blur to stick with the Figma design. You can refer to [this link](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/blur) for how to do it.

| Your Design | Figma Design |
|---------|----------|
| image | image |

There is unnecessary white space at the bottom of the `popup` window in both mobile and desktop views let's remove that to stick more like to the Figma design.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @abiodunraheem,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._



## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @abiodunraheem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**